docs/UserGuide.md

```diff
@@ -59,11 +59,12 @@ Examples:
 
 ### Listing all patients/appointments : `list`
 
-Shows a list of all patients or appointments, depending on the parameter given.
+Shows a list of all patients and/or appointments, depending on the parameter given.
 
 Format:
 * `list patients`
 * `list appts`
+* `list all`
```

Comment:
Perhaps it would be better to include the explanation for each of the command specifically for clarity.

Reply:
Nice suggestion! Sure